lib/galaxy/datatypes/converters/bigbed_to_bed_converter.xml

```diff
@@ -0,0 +1,97 @@
+<tool id="bigbedtobed" name="Convert from bigBed to ascii bed format." version="@TOOL_VERSION@+galaxy@SUFFIX_VERSION@" profile="20.01">
```

Comment:
```suggestion
<tool id="ucsc_bigbedtobed" name="Convert from bigBed to ascii bed format." version="@TOOL_VERSION@+galaxy@SUFFIX_VERSION@" profile="20.01">
```

Reply:
"standard" for choosing ids of ucsc tools (https://github.com/galaxyproject/tools-iuc/tree/main/tools/ucsc_tools)

Reply:
you can also likely copy the citation and more from e.g. https://github.com/galaxyproject/tools-iuc/blob/main/tools/ucsc_tools/ucsc_chainantirepeat/ucsc_chainantirepeat.xml